plugins/osdn/ovs/plugin.go

```diff
@@ -124,25 +125,77 @@ func (plugin *ovsPlugin) getVNID(namespace string) (string, error) {
 	return "0", nil
 }
 
+var minRsrc = resource.MustParse("1k")
+var maxRsrc = resource.MustParse("1P")
+
+func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
+	if rsrc.Value() < minRsrc.Value() {
+		return fmt.Errorf("resource is unreasonably small (< 1kbit)")
```

Comment:
use minRsrc in the error message? otherwise someone's going to change the min/max and forget to change them here too